README.md

```diff
@@ -16,6 +16,14 @@ Reporting Issues & Contributing
 
 Before you email me, please read [Issue Reporting Guideline](CONTRIBUTING.md) carefully. Twice. (Don't email me)
 
+Major changes since 2.0.0
+-------------------------------
+The plugin has been refactored to use in-memory processing of library entries, in contrast to its old version where library jars are unzipped to disk.
+
+This has positive performance implications, especially for large projects, machines with slow disks (i.e. spinning hard drives) or systems with slow file systems (i.e. WSL1 emulated file access).
+
+It has also streamlined the plugin code for contributors.
+
```

Comment:
I suggest we move this part out to a new Google doc or gist, and eventually include it into GitHub Release note.
For reference the release note for sbt-assembly 1.0.0 looked like this - https://github.com/sbt/sbt-assembly/releases/tag/v1.0.0

Along with the new features, we should list out all breakages and document migration path where necessary since sbt-assembly is used by thousands of builds.

Reply:
I will remove it but I don't know how to do a new Google doc or gist yet. Is the code committed as part of this repo?

Reply:
> Is the code committed as part of this repo?

Not sure what you mean here.

Reply:
I mean the release notes :)

Reply:
Yes and no. Release notes https://github.com/sbt/sbt-assembly/releases is part of this _Github repo_, but it's not part of _git repo_ and there's no history tracking etc.